### ISSUE TITLE
Bug or design flaw: The Ruins of Alph research center's roof color at night looks mistaken

### ISSUE THREAD

**Issue by Rangi42** [pret/pokecrystal#805]
This looks like an oversight:

![image](https://user-images.githubusercontent.com/35663410/109438117-634f7700-79f6-11eb-804f-479cd98fe531.png)

The roofs.pal data:

```
; group 3 (dungeons)
	RGB 21,21,21, 11,11,11 ; morn/day
	RGB 21,21,21, 17,08,07 ; nite
```

The morn/day values are the same placeholder grays from group 0 (unused). The nite values look inappropriate at night.

**Comment by ShadowOne333** [comment]
Using Pewter City's night palette (as it uses a similar gray palette for its buildings) gives it a good match with the default morning/day palette already used for Group 3.
It could be a good workaround for it.

Inside gfx/tilesets/roofs.pal:
```
; group 3 (dungeons)
	RGB 21,21,21, 11,11,11 ; morn/day
	RGB 09,09,11, 04,05,07 ; nite
```

![pokecrystal-0](https://user-images.githubusercontent.com/13888698/118909279-07eba180-b8e0-11eb-9f8e-a31094c10c76.png)

**Comment by coltongit** [comment]
Fix in #848
